examples/demo_all_cisco.pp

```diff
@@ -38,6 +38,8 @@
   include ciscopuppet::cisco::demo_pim
   include ciscopuppet::cisco::demo_portchannel
   include ciscopuppet::cisco::demo_snmp
+  #include ciscopuppet::cisco::demo_stp_bd
```

Comment:
Want to understand the conflict a bit more, but can you add a comment to the right to indicate the mutual exclusivity of these two entries?